src/fsdocs-tool/ProjectCracker.fs

```diff
@@ -426,8 +426,11 @@ module Crack =
         let projectFiles =
             projectFiles
             |> List.choose (fun s ->
-                if s.Contains(".Tests") || s.Contains("test") then
+                // Other libraries' test projects will be filtered out by projInfos later on,
+                // but this make sure FSharp.Formatting.ApiDocs test projects get excluded.
+                if s.Contains("tests\\FSharp") then
```

Comment:
Are you sure about this bit?
When I commented out the `|> List.choose` I was able to run all the tests just fine.

(I ran ` dotnet run --project .\build\build.fsproj -- -t Tests` at the root)

Reply:
The tests run fine, the problem is that FSharp.Formatting tests run on regular (non-test) projects such as this one: https://github.com/fsprojects/FSharp.Formatting/blob/main/tests/FSharp.ApiDocs.Tests/files/crefLib1/crefLib1.fsproj

crefLib1 itself is not a test project, so if we do not exclude it explicitly somehow then crefLib1 will be added to FSharp.Formatting's api reference here: https://fsprojects.github.io/FSharp.Formatting/reference/index.html

Reply:
Ah ok, I see. Thanks for clarifying that.
In that case, I would go for a slightly more strict approach like:
```fsharp
            let fsharpFormattingTestProjectMarker = $"FSharp.ApiDocs.Tests{Path.DirectorySeparatorChar}files"

            projectFiles
            |> List.filter (fun s ->
                let isTestProject = s.Contains fsharpFormattingTestProjectMarker

                if isTestProject then
                    printfn
                        $"  skipping project '%s{Path.GetFileName s}' because it is a project part of the FSharp.Formatting test suite."

                not isTestProject)
```

To really put emphasis on those selected projects that we are excluding.
The `Path.DirectorySeparatorChar` would make it work cross-platform, I'm not sure the current solution would.

What do you think?

Reply:
Sure, I just pushed a more strict version based on your suggestion plus the required filtering out of `tests/FSharp.Formatting.TestHelpers/FSharp.Formatting.TestHelpers.fsproj`. The test helpers project will otherwise show up in FSharp.Formatting API docs.